nt-be/src/main.rs

```diff
@@ -21,6 +22,64 @@ async fn main() {
             .expect("Failed to initialize application state"),
     );
 
+    // Spawn background monitoring task
+    let state_clone = state.clone();
+    tokio::spawn(async move {
+        use near_api::Chain;
+        use nt_be::handlers::balance_changes::account_monitor::run_monitor_cycle;
+
+        // Get monitoring interval from env or default to 5 minutes
+        let interval_minutes: u64 = std::env::var("MONITOR_INTERVAL_MINUTES")
```

Comment:
Should we move to env too>?

Reply:
yes. Fixed here: https://github.com/NEAR-DevHub/treasury26/pull/22/changes#diff-f753e2492c946b5bf2682cf271dc95df08c1ee0221f446a7945b0844fe24cdfdR32